ui/js/main.js

```diff
@@ -0,0 +1,11 @@
+window.addEventListener('load', function () {
```

Comment:
@peter-geek We are not going to use Jquery, please remove any Jquery related changes